### ISSUE TITLE
Extended support for AsciiDoc

### ISSUE THREAD

**Issue by michix** [xwmx/nb#259]
As of issue #132 there is support for AsciiDoc. Is it possible to improve this support a bit regarding the following points?

. When creating a new note, the default file extension is `.md`. Is it possible to use `.adoc` here?
. When creating a new note with `nb add --title filename.adoc` a template for a Markdown-Document is used, i.e. the heading is done with `#` instead of `=`.

Probably I missed something in the great documentation. If so, I'm very sorry!

**Comment by xwmx** [comment]
@michix Yes, you can set the default extension to `.adoc` with the following command:

```bash
nb set default_extension "adoc"
```

New notes will then be created with the `.adoc` extension. AsciiDoc titles are also recognized in `ls` / `list`.

The missing `add --title` formatting for AsciiDoc was an oversight. It's now included in the latest versions in the repository and will be in the next release version.

Let me know if there are any other issues you are aware of with this.

**Comment by michix** [comment]
Many thanks! There is one more thing: in markdown the title is formatted using `# Title` while it is in asciidoc `= Title`.

Right now, this is not considered.
